aiofreepybox/api/dhcp.py

```diff
@@ -1,28 +1,100 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
 class Dhcp:
 
     def __init__(self, access):
         self._access = access
 
+    static_lease_schema = {
+        'ip': '',
+        'mac': '',
+        'comment': ''
+    }
+
+    dhcp_configuration_schema = {
+        'alwaysBroadcast': bool,
+        'dns': [''],
+        'enabled': bool,
+        'ipRangeStart': '',
+        'ipRangeEnd': '',
+        'stickyAssign': bool
+    }
+
+    dhcp_v6_configuration_data_schema = {
+        'dns': [''],
+        'enabled': bool,
+        'useCustomDns': bool
```

Comment:
As discussed in previous PRs, use `True` or `False` rather than `bool` in schemas

Reply:
Fixed